pkg/services/sqlstore/migrations/dashboard_public_config_mig.go

```diff
@@ -8,11 +8,9 @@ func addPublicDashboardMigration(mg *Migrator) {
 	var dashboardPublicCfgV1 = Table{
 		Name: "dashboard_public_config",
 		Columns: []*Column{
-			{Name: "uid", Type: DB_BigInt, IsPrimaryKey: true},
```

Comment:
Its not backwards compatible to change a column type - I'd recommend making a new column with the new type and updating the application to write to both for some time, then remove the old column in a future release.

Happy to answer any questions on my motivation here.

Reply:
I agree, since this migration has already been pushed to main (so its in 9.0.0-beta1), we should assume there will be instances that have ran it in the wild. We should just have a new migration script to amend the table.

Reply:
Thanks for catching this! We're going to drop and recreate the table with the correct pkey type